test/Dataworker.buildRoots.ts

```diff
@@ -1,114 +1,102 @@
   RunningBalances,
   V2DepositWithBlock,
   V2FillWithBlock,
   V3DepositWithBlock,
   V3FillWithBlock,
 } from "../src/interfaces";
-import {
-  EMPTY_MERKLE_ROOT,
-  assert,
-  bnZero,
-  compareAddresses,
-  fixedPointAdjustment,
-  getRealizedLpFeeForFills,
-  getRefund,
-  getRefundForFills,
-} from "../src/utils";
-import {
-  CHAIN_ID_TEST_LIST,
-  MAX_L1_TOKENS_PER_POOL_REBALANCE_LEAF,
-  MAX_REFUNDS_PER_RELAYER_REFUND_LEAF,
-  amountToDeposit,
-  destinationChainId,
-  mockTreeRoot,
-  originChainId,
-  refundProposalLiveness,
-  repaymentChainId,
-  sampleRateModel,
-} from "./constants";
+import { assert, bnZero, fixedPointAdjustment } from "../src/utils";
+import { amountToDeposit, destinationChainId, mockTreeRoot, originChainId, repaymentChainId } from "./constants";
 import { setupFastDataworker } from "./fixtures/Dataworker.Fixture";
 import {
   BigNumber,
   Contract,
   SignerWithAddress,
-  buildDeposit,
-  buildFill,
-  buildFillForRepaymentChain,
-  buildPoolRebalanceLeafTree,
-  buildPoolRebalanceLeaves,
-  buildRelayerRefundTreeWithUnassignedLeafIds,
-  buildSlowFill,
-  buildSlowRelayLeaves,
-  buildSlowRelayTree,
   buildV3SlowRelayLeaves,
-  constructPoolRebalanceTree,
-  createSpyLogger,
-  deployNewTokenMapping,
   depositV3,
-  enableRoutesOnHubPool,
   ethers,
   expect,
   fillV3,
-  fillV3Relay,
   getDefaultBlockRange,
-  lastSpyLogIncludes,
   requestSlowFill,
-  setupTokensForWallet,
-  sinon,
   toBN,
-  toBNWei,
   buildV3SlowRelayTree,
-  updateDeposit,
 } from "./utils";
 import { utils as sdkUtils, interfaces } from "@across-protocol/sdk-v2";
 
 // Tested
 import { Dataworker } from "../src/dataworker/Dataworker";
 
 let spokePool_1: Contract, erc20_1: Contract, spokePool_2: Contract, erc20_2: Contract;
-let l1Token_1: Contract, hubPool: Contract, timer: Contract, configStore: Contract;
-let depositor: SignerWithAddress, relayer: SignerWithAddress, dataworker: SignerWithAddress;
+let l1Token_1: Contract;
+let depositor: SignerWithAddress, relayer: SignerWithAddress;
 
-let hubPoolClient: HubPoolClient, configStoreClient: ConfigStoreClient;
+let hubPoolClient: HubPoolClient;
 let dataworkerInstance: Dataworker;
 let spokePoolClients: { [chainId: number]: SpokePoolClient };
 
-let spy: sinon.SinonSpy;
-
 let updateAllClients: () => Promise<void>;
 
 describe("Dataworker: Build merkle roots", async function () {
   beforeEach(async function () {
     const fastDataworkerResult = await setupFastDataworker(ethers);
-    configStoreClient = fastDataworkerResult.configStoreClient as unknown as ConfigStoreClient;
     ({
-      hubPool,
       spokePool_1,
       erc20_1,
       spokePool_2,
       erc20_2,
-      configStore,
       hubPoolClient,
       l1Token_1,
       depositor,
       relayer,
       dataworkerInstance,
-      dataworker,
-      timer,
       spokePoolClients,
-      spy,
       updateAllClients,
     } = fastDataworkerResult);
+    await updateAllClients();
+    await updateAllClients();
```

Comment:
I assume it's not intended to call this twice.

```suggestion
    await updateAllClients();
```